src/background/services/lock/index.ts

```diff
@@ -28,27 +30,30 @@ export default class LockService implements IBackupable {
 
   private unlockCB?: () => void;
 
+  private backupService: BackupService;
+
   private constructor() {
     this.isUnlocked = false;
     this.passwordChecker = "Password is correct";
     this.passwordStorage = new SimpleStorage(PASSWORD_DB_KEY);
     this.password = undefined;
     this.unlockCB = undefined;
+    this.backupService = BackupService.getInstance();
```

Comment:
I don't think it's correct to use backup service inside backupable. You can move `getAuthenticBackupCiphertext` method to crypto service and import here so with don't have cycle dependency between services.